dps/analysis/xsection/02_unfold_and_measure.py

```diff
@@ -120,16 +120,15 @@ def unfold_results( results, category, channel, tau_value, h_truth, h_measured,
     # print "h_unfolded_data bin edges : ", h_unfolded_data
     h_data_no_fakes = h_data_no_fakes.rebinned(2)
     if category == 'central':
-        # Todo add the correlation matrices
-        # Return the covariance matrices
-        covariance_matrix, normalised_covariance_matrix = unfolding.get_covariance_matrix()
+        # Return the covariance matrices (They have been normailsed)
+        covariance_matrix, correlation_matrix = unfolding.get_covariance_matrix()
```

Comment:
I assume "normalised" means that the covariance matrix refers to the normalised cross section?  We may need the covariance matrix before normalisation i.e. straight from TUnfold, for some of our checks.